.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you remove this file

Reply:
Ugh...apologies for letting that creep in.